e2e-tests/e2e-tests.ts

```diff
@@ -0,0 +1,49 @@
+const assert = require('assert')
+const puppeteer = require('puppeteer')
+
+let browser
+let page
+let host = `https://passport-verify-stub-relying-party-${process.env.E2E_TEST_ENVIRONMENT || 'integration'}.cloudapps.digital/`
```

Comment:
I think it would be flexible to provide the full host as an envvar.

Reply:
done